app/views/spree/admin/orders/invoice4.html.haml

```diff
@@ -46,6 +46,8 @@
         %br
         = "#{t :invoice_number}:"
         = @order.display_number
+        - if @order.previous_invoice.present?
+          = "#{t :invoice_cancel_and_replace_invoice} #{ @order.previous_invoice.display_number}"
```

Comment:
I'm not sure if it will matter in other languages, but just in case I'd suggest using interpolation here, just in case some translations want to write something like 
"cancels and replaces number %{canceled_invoice_number} invoice."